github/ci/prow/files/jobs/kubevirt/user-guide/user-guide-postsubmits.yaml

```diff
@@ -0,0 +1,47 @@
+postsubmits:
+  kubevirt/user-guide:
+  - name: push-user-guide-source-build-and-push-to-gh-pages
+    branches:
+    - ^source$
+    annotations:
+      testgrid-create-test-group: "false"
+    decorate: true
+    extra_refs:
+    - org: kubevirt
+      repo: user-guide
+      base_ref: gh-pages
+      path_alias: github.com/kubevirt/user-guide-gh-pages
+    - org: kubevirt
+      repo: project-infra
+      base_ref: master
+    spec:
+      containers:
+      - image: docker.io/library/python:3.7
+        env:
+        - name: GIT_ASKPASS
+          value: ../project-infra/hack/git-askpass.sh
+        - name: GIT_AUTHOR_NAME
+          value: kubevirt-bot
+        - name: GIT_AUTHOR_EMAIL
+          value: rmohr+kubebot@redhat.com
+        command: [ "/bin/sh", "-c" ]
+        args:
+        - |
+          set -e
+          git config --global user.email "$GIT_AUTHOR_EMAIL"
+          git config --global user.name "$GIT_AUTHOR_NAME"
+          commit=$(git show -s --format=%H)
+          make build
+          cd ../user-guide-gh-pages
+          find . -mindepth 1 -maxdepth 1 -regextype egrep -not -regex '\./(\.git.*|\.nojekyll)' -print | xargs rm -rf
```

Comment:
minor nit: while we don't have or plan to have file/dir names that contain spaces, I believe it would not hurt to use null termination for argument separation here for some extra peace of mind:

```suggestion
          find . -mindepth 1 -maxdepth 1 -regextype egrep -not -regex '\./(\.git.*|\.nojekyll)' -print0 | xargs -0 rm -rf
```

Reply:
@codificat The diff doesn't look correct
```
# find . -mindepth 1 -maxdepth 1 -regextype egrep -not -regex '\./(\.git.*|\.nojekyll)' -print0
./mkdocs.yml./update_changelog.sh./Makefile./LICENSE./docs./README.md./Gemfile./site./netlify.conf./.aspell.en.pws./OWNERS./Rakefile#
```
Notice all the newline chars are missing?

Reply:
I think it should be ...
```
find . -mindepth 1 -maxdepth 1 -regextype egrep -not -regex "(\.git.*|\.nojekyll|\./site)" -print | xargs -0 rm -rf
./mkdocs.yml
./update_changelog.sh
./Makefile
./LICENSE
./docs
./.nojekyll
./.gitignore
./README.md
./Gemfile
./netlify.conf
./.git
./.aspell.en.pws
./OWNERS
./Rakefile
```

Reply:
ah I understand now.  +1